src/Components/test/testassets/ComponentsApp.Server/Startup.cs

```diff
@@ -27,7 +29,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             }
 
             app.UseStaticFiles();
-            app.UseRazorComponents<App.Startup>();
+            app.UseRouting(builder =>
+            {
+                builder.MapRazorPages();
+                builder.MapComponentHub<App.App>("app");
```

Comment:
This looks so good now! Awesome job, @javiercn!

Reply:
It does. It’s also fast to load because of the pre-rendering